server/database.js

```diff
@@ -79,6 +79,7 @@ class Database {
         "patch-added-kafka-producer.sql": true,
         "patch-add-certificate-expiry-status-page.sql": true,
         "patch-monitor-oauth-cc.sql": true,
+        "patch-add-cache-bust.sql": true,
```

Comment:
Need to use knex migrations

https://github.com/louislam/uptime-kuma/tree/master/db/knex_migrations